webnn/resources/test_data/reshape.json

```diff
@@ -0,0 +1,272 @@
+{
+  "tests": [
```

Comment:
How about one test case coming from or going to 1D? e.g. `"newShape": [24],`

Reply:
And/or `"newShape: [-1]` which would consolidate all elements into a single 1D tensor. I don't really care which, but I think we should have at least one 1D test case.

Reply:
Thanks, I've added such tests. Please take another look.